src/app/layout.js

```diff
@@ -28,6 +33,10 @@ function Header() {
             <h1 className="text-2xl text-white font-custom-titles w-72 md:w-full">Recursive Ordinals Builder</h1>
           </div>
           <div className="flex-shrink-0">
+            <a>{user ? `Hi, ${user.name}! ` : ""}</a>
```

Comment:
Can you use the SimpleButton component below?

import { SimpleButton } from '../components/buttons.jsx';

<SimpleButton label={isLoading ? "..." : (user ? "Login" : "Logout")} onClick={() => window.open(`/api/auth/${user ? "logout" : "login"})} active={!isLoading} />

Reply:
Done! Note that this requires adding an outline to the button since it's in the header.